homeassistant/components/logger/services.yaml

```diff
@@ -1,6 +1,25 @@
 set_default_level:
   description: Set the default log level for components.
   fields:
-    level: {description: 'Default severity level. Possible values are notset, debug,
-        info, warn, warning, error, fatal, critical', example: debug}
-set_level: {description: Set log level for components.}
+    level:
+      description: "Default severity level. Possible values are notset, debug, info, warn, warning, error, fatal, critical"
+      example: debug
+
+set_level:
+  description: Set log level for components.
+  fields:
+    homeassistant.core:
+      description: "Home Assistant name space severity level. Possible values are notset, debug, info, warn, warning, error, fatal, critical"
+      example: debug
+    homeassistant.components.mqtt:
+      description: "Integration name space severity level. Possible values are notset, debug, info, warn, warning, error, fatal, critical"
+      example: warning
+    homeassistant.components.smartthings.light:
+      description: "Component platform name space severity level. Possible values are notset, debug, info, warn, warning, error, fatal, critical"
+      example: info
+    custom_components.my_integration:
+      description: "Custom component name space severity level. Possible values are notset, debug, info, warn, warning, error, fatal, critical"
+      example: debug
+    aiohttp:
+      description: "Python module name space severity level. Possible values are notset, debug, info, warn, warning, error, fatal, critical"
```

Comment:
I don't like the intro line for these examples, it's super confusing.  Let's do something like:

> Example on how to change the logging level for a Python module. Possible values are …

And then drop "notset". Not necessary to list it.

Reply:
Done and make lot more sense this way